### ISSUE TITLE
Themes Radio Button Group in UX settings should be wrapped in a <fieldset />, not a <div />

### ISSUE THREAD

**Issue by nickytonline** [forem/forem#11243]
**Describe the bug**

When grouping controls, like a radio button group, a `<fieldset />` should be used instead of a generic `<div />`. For more information, see the [fieldset](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) documentation on MDN.

**To Reproduce**

n/a

**Expected behavior**

The radio button group for the site themes, should render the same visually, but be more semantically correct in markup land.

**Screenshots**

![image](https://user-images.githubusercontent.com/833231/97948615-19fc5d00-1d5f-11eb-8c4c-8b12ce29261a.png)

**Desktop (please complete the following information):**

- OS, version: n/a
- Browser, version: n/a

**Smartphone (please complete the following information):**

- Device: n/a
- OS, version: n/a
- Browser, version: n/a

**Additional context**
